src/RealSenseExtractor/GraphanaPublisher.py

```diff
@@ -0,0 +1,44 @@
+# 
+# Copyright (C) 2025 Intel Corporation. 
+# 
+# SPDX-License-Identifier: Apache-2.0 
+#
+
+from Publisher import Publisher
+from influxdb_client import InfluxDBClient, Point
+from influxdb_client.client.write_api import SYNCHRONOUS
+from datetime import datetime
+import json
+
+class GraphanaPublisher(Publisher):
```

Comment:
Please fix the spelling of Grafana in the file names, files, etc where applicable.
```suggestion
class GrafanaPublisher(Publisher):
```

Reply:
Updated the file names.